app/modules/booking/bookTickets/authhelper.js

```diff
@@ -0,0 +1,36 @@
+const jwt = require('jsonwebtoken')
+const config = require('../../../../config/config')
+
+function verifytoken(req, res, next) {
+
+    const bearerHeader = req.headers['authorization'];
+    if (typeof bearerHeader !== 'undefined') {
```

Comment:
just need to check if(bearerHeader), if it is undefined then the else case will get executed.

Should verify there is any unwanted comment code through the application before placing a PR

Reply:
File naming convention should be auth.helper as standard